lib/bolt/transport/sudoable.rb

```diff
@@ -118,16 +118,9 @@ def run_task(target, task, arguments, options = {})
                 execute_options[:environment] = envify_params(arguments)
               end
 
-              if conn.run_as && stdin
-                # Inject interpreter in to wrapper script and remove from execute options
-                wrapper = make_wrapper_stringio(remote_task_path, stdin, execute_options[:interpreter])
-                execute_options.delete(:interpreter)
-                remote_wrapper_path = conn.write_executable(dir, wrapper, 'wrapper.sh')
-                command << remote_wrapper_path
-              else
-                command << remote_task_path
-                execute_options[:stdin] = stdin
-              end
+              command << remote_task_path
```

Comment:
I think the `command` variable can go away entirely. It appears that now it starts out as an empty array and we put `remote_task_path` in it and nothing else.